content/en/blog/_posts/2022-05-06-storage-capacity-GA/index.md

```diff
@@ -56,12 +56,12 @@ Also not solved is support in Cluster Autoscaler for Pods with volumes. For CSI
 drivers with storage capacity tracking, a prototype was developed and discussed
 in [a PR](https://github.com/kubernetes/autoscaler/pull/3887). It was meant to
 work with arbitrary CSI drivers, but that flexibility made it hard to configure
-and slowed down scale up operations: because autoscaler was unable to simulate
+and slowed downscale up operations: because autoscaler was unable to simulate
 volume provisioning, it only scaled the cluster by one node at a time, which
 was seen as insufficient.
 
 Therefore that PR was not merged and a different approach with tighter coupling
-between autoscaler and CSI driver will be needed. For this a better
+between autoscaler and CSI driver will be needed. For this, a better
```

Comment:
I think this addition is OK, except that adding just a comma to a blog article is a [trivial change](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits) that wouldn't be worth making.

Reply:
Thank you for your valuable feedback @sftim